playerbot/strategy/paladin/PaladinAiObjectContext.cpp

```diff
@@ -651,9 +666,6 @@ namespace ai
             static Action* repentance_on_enemy_healer(PlayerbotAI* ai) { return new CastRepentanceOnHealerAction(ai); }
             static Action* holy_shock(PlayerbotAI* ai) { return new CastHolyShockAction(ai); }
             static Action* holy_shock_on_party(PlayerbotAI* ai) { return new CastHolyShockOnPartyAction(ai); }
-            static Action* update_pve_strats(PlayerbotAI* ai) { return new UpdatePaladinPveStrategiesAction(ai); }

```

Comment:
Careful, you have removed this actions, this is important

Reply:
No i didnt
![obraz](https://user-images.githubusercontent.com/28183654/236528320-40675b70-e61c-423f-8121-5e67d6941c45.png)

Reply:
Fixed